### PR TITLE
Cleans up comments, consistent styling for function and param names

### PR DESCRIPTION
**Pull request by akhila-raju**
## PR Description
Cleans up comments, consistent styling for function and param names to match spec.

### CODE REVIEW

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 1]
```diff
@@ -110,11 +108,12 @@ public static BeaconStateWithCache get_genesis_beacon_state(
...
             ValidatorsUtil.get_active_validator_indices(
                 state.getValidator_registry(), UnsignedLong.valueOf(GENESIS_EPOCH)));
-    for (Bytes32 root : state.getLatest_active_index_roots()) {
-      root = genesis_active_index_root;
+    for (int index = 0; index < state.getLatest_active_index_roots().size(); index++) {
+      state.getLatest_active_index_roots().set(index, genesis_active_index_root);
     }
+
     state.setCurrent_shuffling_seed(generate_seed(state, UnsignedLong.valueOf(GENESIS_EPOCH)));
-    state.setDeposit_index(UnsignedLong.valueOf(genesis_validator_deposits.size()));
```
Comment (cemozerr) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:117]: Why are we removing this?

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 2]
```diff
@@ -573,11 +572,10 @@ public static void initiate_validator_exit(BeaconState state, int index) {
   public static void exit_validator(BeaconState state, int index) {
     Validator validator = state.getValidator_registry().get(index);
 
-    UnsignedLong delayed_activation_exit_epoch = get_delayed_activation_exit_epoch(get_current_epoch(state));
+    UnsignedLong delayed_activation_exit_epoch =
+        get_delayed_activation_exit_epoch(get_current_epoch(state));
     // The following updates only occur if not previous exited
-    if (validator.getExit_epoch().compareTo(delayed_activation_exit_epoch) <= 0) {
```
Comment (cemozerr) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:578]: This change is not in line with spec

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java [hunk 3]
```diff
@@ -1051,12 +1052,9 @@ public static UnsignedLong max(UnsignedLong value1, UnsignedLong value2) {
    *     - Spec v0.4</a>
    */
   public static UnsignedLong get_domain(Fork fork, UnsignedLong epoch, int domain_type) {
-    return bytes_to_int(
```
Comment (cemozerr) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:1054]: This change is not in line with spec.

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java [hunk 4]
```diff
@@ -99,23 +101,13 @@ public static UnsignedLong get_previous_total_balance(BeaconState state) {
...
-    List<Integer> attester_indices = new ArrayList<>();
-    for (List<Integer> validator_index_set : validator_index_sets) {
-      for (Integer validator_index : validator_index_set) {
-        if (!attester_indices.contains(validator_index)) {
-          attester_indices.add(validator_index);
-        }
-      }
+    HashSet<Integer> output = new HashSet<>();
+    for (PendingAttestation a : attestations) {
+      output.addAll(get_attestation_participants(state, a.getData(), a.getAggregation_bitfield()));
```
Comment (cemozerr) [inline ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java:106]: Just to check: are you 100% that addAll doesn't create duplicates in the hashset here?

**Review by cemozerr — CHANGES_REQUESTED** [review]

File: ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java (hunk 4 above)
Comment (akhila-raju) [reply ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java:106]: Yes, HashSets do not allow duplicates.

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java (hunk 2 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:578]: It's the same logic with fewer lines of code.

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java (hunk 1 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:117]: It's not in line with the spec.

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java (hunk 3 above)
Comment (akhila-raju) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java:1054]: Reverted back.

**Review by cemozerr — APPROVED** [review]
LGTM.

**Pull request merged**